src/foundry/client/pixi/extensions/rectangle.d.mts

```diff
@@ -217,32 +216,39 @@ declare module "pixi.js" {
   }
 
   namespace Rectangle {
-    /** @internal Helper type for interface to simplify InexactPartial usage */
-    type _IntersectPolygonOptions = InexactPartial<{
-      /** The number of points which defines the density of approximation */
-      density: number;
+    type CS_ZONES = Brand<number, "PIXI.Rectangle.CS_ZONES">;
 
-      /** The clipper clip type */
-      clipType: number;
+    /** @remarks Options for `weilerAtherton: true` (or not provided) */
+    interface WACIntersectPolygonOptions extends WeilerAthertonClipper._CombineOptions {
+      weilerAtherton?: true;
+    }
+
+    /**
+     * @remarks Options for `weilerAtherton: false`. Only `clipType` has NullishProps applied,
```

Comment:
Maybe duplicate `clipType` so we can document on the property directly? I'm hopeful it'd show up in intellisense.